test/src/test/java/jenkins/model/JenkinsTest.java

```diff
@@ -317,6 +317,7 @@ public void testDoScript() throws Exception {
         wc.withBasicApiToken(User.getById("bob", true));
         wc.assertFails("script", HttpURLConnection.HTTP_FORBIDDEN);
 
+        //TODO: remove once RUN_SCRIPTS is finally retired
```

Comment:
It is unclear why the permission above was changed, and why this comment is necessary.

Reply:
The comment is to remove the step of the test when `RUN_SCRIPTS` is fully removed as that check will not make sense

Reply:
> changed back … leave the test as-is for now

But then this PR wouldn't show a difference…?

Reply:
Ah, now I get it.

It probably makes sense to convert this test to demonstrate that granting `RUN_SCRIPTS` does not allow accessing the script console anymore.

Reply:
> It probably makes sense to convert this test to demonstrate

… which is exactly what this does now 🤦‍♂